frontend/src/components/option/ExtraOptionSelector.tsx

```diff
@@ -44,7 +82,13 @@ function ExtraOptionSelector({ optionList, handleClickOptionCard }: Props) {
         <OptionCard.Extra
           key={opt.id}
           info={opt}
-          isChecked={hasData({ id: opt.id, name: opt.name, price: opt.price, image: opt.image })}
+          isChecked={hasData({
+            id: opt.id,
+            name: opt.name,
+            price: opt.price,
+            image: opt.image,
+            isPackage: opt.isPackage,
+          })}
```

Comment:
```suggestion
          isChecked={hasData({
               ...opt
          })}
```

opt에 있는 모든값을 풀어주는거면 요렇게 해도 좋을듯!

Reply:
저기에 있는 Props가  Omit<OptionInfo, "isQuotation"> 이거라서 opt에서 isQuotation만 빼야하는데,, 스프레드 활용할 수 있어??